Automattic/woocommerce-payments
# Add JCB capability request notice

Fixes https://github.com/Automattic/woocommerce-payments/issues/6804

#### Changes proposed in this Pull Request
Add a notice to request JCB capability above Payment Methods.

<img width="679" alt="Screenshot 2023-09-22 at 0 43 53" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/ef5a7070-943a-463d-ad13-14a8e0cd5f64">

#### Testing instructions
*Case 1*
- Connect a Japanese account.
- Navigate to `Payments` -> `Settings`.
- Scroll down to Payment Methods.
- Verify the notice is displayed above Payment Methods.
- Click the `Enable JCB` button.
- Verify the notice disappears.
- Refresh the page and navigate to Payment Methods.
- Verify the capability is requested and the notice text is changed.
<img width="659" alt="Screenshot 2023-09-22 at 0 48 10" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/d2fd25b3-93e3-4cc9-8a4d-41c7f32c7431">

*Case 2*
- Connect a Japanese account.
- Navigate to `Payments` -> `Settings`.
- Scroll down to Payment Methods.
- Verify the notice is displayed above Payment Methods.
- Click the dismiss icon on the top right corner.
- Verify a modal is displayed warning you the notice won't be shown again
- Click `Yes, continue`
- Verify the notice is hidden.
- Refresh the page
- Verify the notice is not displayed

<img width="563" alt="Screenshot 2023-09-22 at 0 49 51" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/774e8438-84a3-46d8-9253-d942197a9e8f">


-------------------


**Post merge**
